ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java

```diff
@@ -1969,6 +1988,32 @@ public void close() throws IOException {
     dynamicVars.clear();
   }
 
+  private static void cleanUpAllSessionStates() {
+    ExecutorService cleanupExecutor = Executors.newCachedThreadPool();
+    try {
+      CompletableFuture<Void> allCleanupTasks = CompletableFuture.allOf(sessionStates.stream()
+              .map(sessionState -> CompletableFuture.runAsync(() -> {
+                  try {
+                      LOG.info("Closing session state: {}", sessionState.getSessionId());
+                      sessionState.close();
+                  } catch (IOException e) {
+                      throw new RuntimeException(e);
```

Comment:
minor: @Indhumathi27, seems that `CompletionException` is recommended here instead of RuntimeException.